EjercicioClases/Model/User/Seller.pm

```diff
@@ -3,6 +3,7 @@ package Model::User::Seller;
 use strict;
 use warnings;
 use Data::Dumper;
+use DBI;
```

Comment:
You may want to remove this use clause since you're actually not using DBI module here.